src/state/slices/opportunitiesSlice/resolvers/foxfarming.ts

```diff
@@ -70,14 +70,20 @@ export const foxFarmingLpMetadataResolver = async ({
   })
   const apy = bnOrZero(calculatedApy).div(100).toString()
   const reserves = await uniV2LPContract.getReserves()
+
+  // Getting the ratio of the LP token for each asset
+  const totalSupply = (await uniV2LPContract.totalSupply()).toString()
+  const foxReserves = bnOrZero(bnOrZero(reserves[1].toString()).toString())
+  const ethReserves = bnOrZero(bnOrZero(reserves[0].toString()).toString())
```

Comment:
See https://docs.uniswap.org/protocol/V2/reference/smart-contracts/pair#getreserves, our tuple is `[ethAssetId, foxAssetId]` since `token0` on the contract is ETH and `token1` is FOX.

https://v2.info.uniswap.org/pair/0x470e8de2ebaef52014a47cb5e6af86884947f08c

We might want to reverse the tuple to honor the contract?

Reply:
as long as we settle/standardize a way of doing this, i'm ok with it. the contract is probably the best source of truth.

Reply:
866b88881069d521fdda2097c89a2bc4837d37f3